PiwikPlugin.inc.php

```diff
@@ -22,8 +22,8 @@ class PiwikPlugin extends GenericPlugin {
 	 * @return boolean True iff plugin initialized successfully; if false,
 	 * 	the plugin will not be registered.
 	 */
-	function register($category, $path) {
-		$success = parent::register($category, $path);
+	function register($category, $path, $mainContextId = null) {
+		$success = parent::register($category, $path, $mainContextId);
```

Comment:
This change is good, but the new parameter isn't included in the function self-documentation. Rather than duplicating the self-docs in every plugin, better to just swap it out for
```
/**
 * @copydoc Plugin::register()
 */
```
...unless the plugin's self-docs actually add something new. (This one doesn't.)

Reply:
Thanks @asmecher. Done.